.drone.star

```diff
@@ -724,6 +725,28 @@ def checkTestSuitesInExpectedFailures(ctx):
         },
     }]
 
+def checkGherkinLint(ctx):
+    return [{
+        "kind": "pipeline",
+        "type": "docker",
+        "name": "check-gherkin-standard",
+        "steps": [
+            {
+                "name": "lint-feature-files",
+                "image": OC_CI_NODEJS % DEFAULT_NODEJS_VERSION,
+                "commands": [
+                    "npm install -g @gherlint/gherlint@1.1.0",
+                    "gherlint -c tests/acceptance/config/.gherlintrc.json tests/acceptance/features",
```

Comment:
i see we have make command for it. cant we just use it instead of this? @saw-jan @prashant-gurung899

Reply:
I think should install the package anyhow before using it in drone. I'm not sure. @saw-jan ?